gulp/docs.js

```diff
@@ -74,74 +69,185 @@ module.exports = {
 				if (!isInComponents) { return acc; }
 
 				var componentName = extractComponentName(file);
+				var exportName;
 
-				var definitionMap;
-				var isPrivateComponent = false
-				var exportIdentiferName;
 				var componentSource = fs.readFileSync(file)
 				var docs = reactDocgen.parse(
 					componentSource,
-					// Resolver
-					function (ast, recast) {
-						definitionMap = {};
+					function resolver(ast, recast) {
+						var component;
+						var childComponents = [];
+						var importLocalNames = [];
+
 						recast.visit(ast, {
-							visitObjectExpression: function (path) {
-								_.forEach(path.get('properties').value, function (property) {
-									if (property.key.name === '_lucidIsPrivate') {
-										isPrivateComponent = true;
-									}
-
-									if (property.key.name === 'render') {
-										var identifier = findParentNodeIdentifier(path);
-										if (identifier) {
-											definitionMap[identifier] = path;
-										}
-									}
-								});
+							visitImportDefaultSpecifier: function(path) {
+								importLocalNames.push(path.value.local.name);
 								return false;
 							},
-							visitExportDefaultDeclaration: function (path) {
-								exportIdentiferName = path.value.declaration.name;
+						});
+
+						recast.visit(ast, {
```

Comment:
nah, because the order matters, i.e. i need `importLocalNames` to be fully populated before I can check if a refernce is to an imported component on L118